yt_dlp/__init__.py

```diff
@@ -980,6 +986,19 @@ def _real_main(argv=None):
             traceback.print_exc()
             ydl._download_retcode = 100
 
+        if opts.list_impersonate_targets:
+            available_targets = ydl.get_available_impersonate_targets()
```

Comment:
add some hints for what targets are supported. 

```
[info] Available impersonate targets
Client  Version OS      OS Version Source
chrome  101     windows 10         curl_cffi
chrome  104     windows 10         curl_cffi
chrome  107     windows 10         curl_cffi
chrome  110     windows 10         curl_cffi
chrome  99      android 12         curl_cffi
chrome  99      windows 10         curl_cffi
edge    101     windows 10         curl_cffi
edge    99      windows 10         curl_cffi
safari  15.3    macos   11         curl_cffi
safari  15.5    macos   12         curl_cffi
firefox -       -       -          tls-client (not installed)
chrome  -       -       -          tls-client (not installed)
firefox -       -       -          curl_cffi (not installed
```

and add some colours to gray out the ones that are not installed